spec/fixtures/.scheme.yml

```diff
@@ -0,0 +1,5 @@
+envs:
```

Comment:
I recommend that instead of duplicating the type name, you declare it once and below it you list the name of those variables with that type.

This is to avoid duplication... :)